.gitmodules

```diff
@@ -10,3 +10,12 @@
 [submodule "components/esp-rainmaker"]
 	path = components/esp-rainmaker
 	url = https://github.com/espressif/esp-rainmaker.git
+[submodule "components/esp32-audio-player"]
+	path = components/esp32-audio-player
+	url = https://github.com/chmorgan/esp32-audio-player.git
+[submodule "components/esp32-file-iterator"]
+	path = components/esp32-file-iterator
+	url = https://github.com/chmorgan/esp32-file-iterator.git
+[submodule "components/esp32-libhelix-mp3"]
+	path = components/esp32-libhelix-mp3
+	url = https://github.com/chmorgan/esp32-libhelix-mp3.git
```

Comment:
Checked the license, it looks like its compatible as the license is quite permissible. Many of the terms appear intended to prevent patents to be applied ontop of the RealNetworks IP (libhelix-mp3).